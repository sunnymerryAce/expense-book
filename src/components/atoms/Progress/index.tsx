import React from "react";
import { CircularProgress } from "@material-ui/core";

const Progress: React.FC = () => {
  return <CircularProgress />;
};

export default Progress;
