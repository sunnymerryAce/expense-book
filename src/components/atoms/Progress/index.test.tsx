import React from "react";
import Progress from ".";
import { shallow } from "enzyme";

describe("<Progress />", () => {
  let container: any = null;

  beforeEach(() => {
    container = shallow(<Progress />);
  });

  afterEach(() => {
    container.unmount();
    container = null;
  });

  test("should match the snapshot", () => {
    expect(container.html()).toMatchSnapshot();
  });
});
