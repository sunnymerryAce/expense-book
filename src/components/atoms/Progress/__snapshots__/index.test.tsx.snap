// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Progress /> should match the snapshot 1`] = `"<div class=\\"MuiCircularProgress-root MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate\\" style=\\"width:40px;height:40px\\" role=\\"progressbar\\"><svg class=\\"MuiCircularProgress-svg\\" viewBox=\\"22 22 44 44\\"><circle class=\\"MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate\\" cx=\\"44\\" cy=\\"44\\" r=\\"20.2\\" fill=\\"none\\" stroke-width=\\"3.6\\"></circle></svg></div>"`;
